google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/PostgreSQLStatementParser.java

```diff
@@ -136,23 +135,6 @@ String removeCommentsAndTrimInternal(String sql) {
     return res.toString().trim();
   }
 
-  String parseDollarQuotedString(String sql, int index) {
```

Comment:
This and the other methods here that appear to be deleted, are just moved to `AbstractStatementParser`, which is the super class of this class. That means that functionally, nothing changes.